packages/plugin-core/src/commands/OpenLink.ts

```diff
@@ -25,47 +27,59 @@ export class OpenLinkCommand extends BasicCommand<CommandOpts, CommandOutput> {
   async execute() {
     const ctx = DENDRON_COMMANDS.OPEN_LINK;
     this.L.info({ ctx });
+
+    let text = "";
+
     if (!isAnythingSelected()) {
-      const error = DendronError.createFromStatus({
-        status: ERROR_STATUS.INVALID_STATE,
-        message: `nothing selected`,
-      });
-      this.L.error({ error });
-      return { error };
+      window.showInformationMessage(
```

Comment:
don't forget to return here

Reply:
The proposed functionality would allow a user to follow a link without selecting it so having nothing selected should no longer be an error